api/src/auth/api.py

```diff
@@ -58,6 +58,7 @@ def login():
 # Define our callback function to check if a token has been revoked or not
 @jwt.token_in_blacklist_loader
 def check_if_token_revoked(decoded_token):
+    return False # for testing purpose
```

Comment:
It doesn't seem like this line should be left in the code.

If we need to override this method for testing, can we use a mock? For example, https://docs.pytest.org/en/latest/monkeypatch.html

Reply:
Sorry about that. I will double check next time to make sure code like this does not get committed. I was just doing some quick-and-dirty testing using an interactive API tester, and somehow forgot to remove this line... I will remove it in a new commit.

Reply:
No worries -- it's easy to miss these small changes.